src/main/controllers/SummaryOfPublicationsController.ts

```diff
@@ -26,10 +24,7 @@ export default class SummaryOfPublicationsController {
                     req.lng === 'cy' ? locationMetadata.welshCautionMessage : locationMetadata.cautionMessage;
             }
 
-            const publications = await summaryOfPublicationsService.getPublications(
```

Comment:
I think we should keep the the parseInt in some way here, as it adds a level of validation on the input data via the query param. Similar comment on the RemoveListSearchResultsController below.

Potentially better as an if statement that throws an error if not an int?